data_test.go

```diff
@@ -63,6 +64,42 @@ func TestRollingChecksum(t *testing.T) {
 
 }
 
+func TestNormalVsFastfpgen(t *testing.T) {
+	fmt.Println("==TestNormalVsFastfpgen==\n")
+
+	blksz := 1024
+	basefile := "testdata/largebinaryfile"
+	//basefile := "/Users/msingh/Downloads/golang.mp4"
```

Comment:
Unintentional. 
I will be cleaning up and add documentation soon.
BTW, If you are using the library, I would recommend to use the master branch build.